### PR TITLE
Add Hexo detection

### PR DESCRIPTION
**Pull request by h404bi**
About Hexo:
- [Site](https://hexo.io)
- [GitHub](https://github.com/hexojs/hexo)
- [Showcases](https://github.com/hexojs/awesome-hexo#showcases)

### CODE REVIEW

File: src/apps.json [hunk 1]
```diff
@@ -3918,6 +3918,19 @@
...
+    "Hexo": {
+      "cats": [
+        "11"
+      ],
+      "icon": "Hexo.png",
+      "html": [
+        "by <a href=\"https?://hexo\\.io/?\"[^>]*>Hexo</"
+      ],
+      "meta": {
+        "generator": "Hexo"
```
Comment (jvoisin) [inline src/apps.json]: What about `"generator": "^Hexo$"` instead?

File: src/apps.json [hunk 2]
```diff
@@ -3918,6 +3918,19 @@
       "script": "hellobar\\.js",
       "website": "http://hellobar.com"
     },
+    "Hexo": {
+      "cats": [
+        "11"
+      ],
+      "icon": "Hexo.png",
+      "html": [
+        "by <a href=\"https?://hexo\\.io/?\"[^>]*>Hexo</"
```
Comment (jvoisin) [inline src/apps.json]: I think that `"Powered by <a href=\"https?://hexo\\.io/?\"[^>]*>Hexo</"` is prettier :3

Comment (h404bi) [reply src/apps.json]: I dont know if it could match lowercase `powered by`, so I omit `Powered`.

File: src/apps.json (hunk 1 above)
Comment (h404bi) [reply src/apps.json]: Use `^Hexo$` couldn't match below cases, I think:
```
<meta generator="Hexo 3.4.4">
<meta generator="Hexo v3.4.4">
```

Comment (jvoisin) [reply src/apps.json]: Please add a way to get the version then ♥

File: src/apps.json (hunk 2 above)
Comment (jvoisin) [reply src/apps.json]: Regexp are case-insensitives :)

File: src/apps.json (hunk 1 above)
Comment (h404bi) [reply src/apps.json]: Added.

**Comment by AliasIO** [comment]
Thanks!

**Pull request merged**